SerialPrograms/Source/PokemonSV/Programs/ShinyHunting/PokemonSV_ShinyHunt-Scatterbug.cpp

```diff
@@ -204,7 +204,6 @@ void ShinyHuntScatterbug::program(SingleSwitchProgramEnvironment& env, BotBaseCo
             send_program_status_notification(env, NOTIFICATION_STATUS_UPDATE);
         }catch(ProgramFinishedException&){
             GO_HOME_WHEN_DONE.run_end_of_program(context);
-            send_program_finished_notification(env, NOTIFICATION_PROGRAM_FINISH);

```

Comment:
So this one isn't actually redundant since it returns instead of throws. So the inconsistency here with the one below is that one throws while the other returns.

Probably better to make them both rethrow the exception.